tests/test_integration.py

```diff
@@ -160,6 +161,23 @@ def check_vm_absent(self, hv_name=None):
                     )
 
 
+class HypervisorCpuThresholdsTest(IGVMTest):
```

Comment:
Inheritance from IGVMTest is not really necessary, as we're not building any VMs in this test, we only check config consistency.

Reply:
Good point. I removed the IGVMTesting

Reply:
Is it correct to pass TestCase instead of IGVMTest? 
I think it is but would like a confirmation

Reply:
`TestCase` is correct